resources/win/win32-open-file-extensions.json

```diff
@@ -0,0 +1,227 @@
+[
+"scpt",
```

Comment:
What do you think about indenting these two spaces and also sorting this list alphabetically to make future additions easier?